src/tests/PhysicsTest.cpp

```diff
@@ -109,7 +109,7 @@ TEST_F(PhysicsManagerTest, JoinCompound) {
             Magnum::Matrix4::rotationX(Magnum::Math::Rad<float>(-1.56)) *
             Magnum::Matrix4::rotationY(Magnum::Math::Rad<float>(-0.25))};
         float boxHeight = 2.0 + (o * 2);
-        Magnum::Vector3 initialPosition{0.0, boxHeight, 0.0};
+        Magnum::Vector3 initialPosition{0.0, float(boxHeight + 1.25), 0.0};
```

Comment:
```suggestion
        Magnum::Vector3 initialPosition{0.0, boxHeight + 1.25f, 0.0};
```

achieves the same without temporarily going through doubles ;)